### ISSUE TITLE
Filenames replaced by title from MP4 metadata on Android in v5+

### ISSUE THREAD

**Issue by MattDetroit** [UniversalMediaServer/UniversalMediaServer#490]
Filenames are being replaced with show name or title but from the metadata of MP4 files (did not notice issue with AVI files), making it appear that episode number and name are removed.
Prettify filename is switched off, when this occurring

For instance the files:
2 Broke Girls-S03E05-And the Cronuts.mp4
2 Broke Girls-S03E06-And the Piece of Sheet.mp4
2 Broke Girls-S03E07-And the Girlfriend Experience.mp4
All become:
2 Broke Girls
2 Broke Girls
2 Broke Girls

Issue occurs with both 5.0 and 5.01, Java 7.
Does not occur for 4.4 nor 4.3.1, Java 7

Problem starts when accessing a folder through an android client (with or without cache enabled)
If cache is enabled, IOS will follow suit (after the problem occurs with the Android client). Won't resolve on IOS until cache is cleared.

The IOS client used during testing is nPlayer, on an iPhone 5S
The Android client used during testing is Vimu on an Amazon Fire TV Stick.

I've attached notes with times to reference in regards to the uploaded logs, in the GItHub attachment thread as well as the original  discussion thread.
http://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=2963

**Comment by SubJunk** [comment]
@MattDetroit thanks for creating this issue

@skeptical it seems based on the logs that we are sending the correct title for these files with "dc:title", for example:

```
&lt;item id="0$1$42$4" parentID="0$1$42" restricted="true"&gt;&lt;dc:title&gt;Last Man Standing-S01E05-Co-Ed Softball&lt;/dc:title&gt;&lt;res xmlns:dlna="urn:schemas-dlna-org:metadata-1-0/" protocolInfo="http-get:*:video/mp4:DLNA.ORG_OP=01" size="447249561" duration="00:44:49.05" resolution="720x406" bitrate="170240" nrAudioChannels="2" sampleFrequency="48000"&gt;http://192.168.0.108:5001/get/0$1$42$4/Last+Man+Standing-S01E05-Co-Ed+Softball.mp4&lt;/res&gt;&lt;upnp:albumArtURI xmlns:dlna="urn:schemas-dlna-org:metadata-1-0/" dlna:profileID="JPEG_TN"&gt;http://192.168.0.108:5001/get/0$1$42$4/thumbnail0000Last+Man+Standing-S01E05-Co-Ed+Softball.mp4&lt;/upnp:albumArtURI&gt;&lt;upnp:albumArtURI xmlns:dlna="urn:schemas-dlna-org:metadata-1-0/" dlna:profileID="JPEG_SM"&gt;http://192.168.0.108:5001/get/0$1$42$4/thumbnail0000Last+Man+Standing-S01E05-Co-Ed+Softball.mp4&lt;/upnp:albumArtURI&gt;&lt;dc:date&gt;2011-11-02T05:10:22&lt;/dc:date&gt;&lt;upnp:class&gt;object.item.videoItem&lt;/upnp:class&gt;&lt;/item&gt;
```

but that file does have just "Last Man Standing" as its title tag, so it seems the displayed title is coming from that tag.

Maybe the code that causes this is similar to the code that allows the renderer to report its name to us and overwrite it on our Status tab, could it be doing that for files by reading their Title tags? I never liked that anyway, especially since my TV reports an ugly name in all caps, so I'd prefer it if we just didn't give the renderers the ability to overrule our names at all.

**Comment by MattDetroit** [comment]
Last Man Standing may be an incorrect example as based on the notes it was brought up with 4.4 and did not experience the issue.  Hannibal would be a better example.5.01 and displayed incorrectly.

**Comment by SubJunk** [comment]
@MattDetroit ah I see, well now it makes more sense because the logs show we are sending the incorrect title for Hannibal at line 16062. Should be easy to fix now.

**Comment by SubJunk** [comment]
Thanks for the reports, it's fixed in 6ae9a42 :)
